content/blogs/2024-06-04-release-0-17.md

```diff
@@ -0,0 +1,599 @@
+3. **Seamlessly Integrated**: you can combine the `PushFlows` and `SyncFlows` tasks together to create a complete Git workflow: push your flows from a development environment to a Git repository and then sync them back to your Kestra environment after they've been reviewed and merged to a production branch.
+4. **Easily Testable**: you can validate your Git workflows in a dry-run mode before committing and pushing your changes.
+
+### PushFlows Task
+
+The PushFlows task allows you to easily commit and push your saved flows to a Git repository. Check the [following documentation](https://kestra.io/docs/how-to-guides/pushflows) and the video demonstration below to learn more about how you can use this task to automate your Git workflow.
+
+<div class="video-container">
+  <iframe src="https://www.youtube.com/embed/OPlNKQZFeho?si=ZvRQfLjnhjDYk1qN" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" referrerpolicy="strict-origin-when-cross-origin" allowfullscreen></iframe>
+</div>
+
+
+### SyncFlows Task
+
+The SyncFlows task automatically checks for changes in your Git branch and deploys them to your Kestra namespace(s), keeping your Kestra environment in sync with your Git repository.
+
+It eliminates the need for CI/CD pipelines — you can use it to sync flows from Git to Kestra on a regular cadence (e.g. an hourly or daily `Schedule` trigger) or whenever changes are merged into a specified Git branch (e.g. a `Webhook` trigger).
+
+<div class="video-container">
+  <iframe width="560" height="315" src="https://www.youtube.com/embed/YbIuqYWLrpA?si=OChhyF1Lz6j8ybGX" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" referrerpolicy="strict-origin-when-cross-origin" allowfullscreen></iframe>
+</div>
+
+**Example: Scheduled Sync**
+
+Sync flows from a Git repository to a Kestra `git` namespace every hour:
+
+```yaml
+id: sync_flows_from_git
+namespace: system
+
+tasks:
+  - id: git
+    type: io.kestra.plugin.git.SyncFlows
+    gitDirectory: flows
+    targetNamespace: git
+    includeChildNamespaces: true # optional; by default, it's set to false to allow explicit definition
+    delete: true # optional; by default, it's set to false to avoid destructive behavior
+    url: https://github.com/anna-geller/flows
+    branch: develop
+    username: anna-geller
+    password: "{{ secret('GITHUB_ACCESS_TOKEN') }}"
+    dryRun: false
+
+triggers:
+  - id: hourly
+    type: io.kestra.plugin.core.trigger.Schedule
+    cron: "0 * * * *"
+```
+
+::alert{type="info"}
+The `PushNamespaceFiles` and `SyncNamespaceFiles` tasks work analogically to the `PushFlows` and `SyncFlows` tasks, but applied to [namespace files](../docs/08.developer-guide/03.namespace-files.md).
```

Comment:
PushNamespaceFiles video: https://youtu.be/-bEnwR5t7VI?si=gOETA5veOTBqjK8J

SyncNamespaceFiles video: https://youtu.be/AbxaDtINcr8?si=-i2qJ7-gWcAokgQJ